### PR TITLE
block modal: tooltips on info buttons

### PR DESCRIPTION
**Pull request by jakob223**
﻿The other half of this ticket (no cancel button, normal modal X) I already did in https://github.com/keybase/client/pull/21370

cc @keybase/y2ksquad @cecileboucheron

### CODE REVIEW

File: shared/chat/blocking/block-modal/index.tsx [hunk 1]
```diff
@@ -59,7 +59,9 @@ const CheckboxRow = (props: CheckboxRowProps) => {
         />
         <Kb.Box style={styles.iconBox} />
         {props.info && !infoShowing && (
-          <Kb.Icon type="iconfont-question-mark" color="grey" onClick={() => setInfoShowing(true)} />
+          <Kb.WithTooltip tooltip={props.info}>
+            <Kb.Icon type="iconfont-question-mark" color="grey" onClick={() => setInfoShowing(true)} />
```
Comment (cecileboucheron) [inline shared/chat/blocking/block-modal/index.tsx]: should this still get an onClick event? Since we want the tooltip to show up on hover.

Comment (jakob223) [reply shared/chat/blocking/block-modal/index.tsx]: I messaged you asking that. Sounds like the answer is no, will do.

Comment (jakob223) [reply shared/chat/blocking/block-modal/index.tsx]: fixed 🔧

File: shared/chat/blocking/block-modal/index.tsx [hunk 2]
```diff
@@ -43,28 +43,24 @@ type CheckboxRowProps = {
...
+      color="red"
+      gapSize={Styles.globalMargins.tiny}
+      label={props.text}
+      on={props.checked}
+      onClick={() => props.onCheck(!props.checked)}
+      style={styles.shrink}
+    />
+    <Kb.Box style={styles.iconBox} />
+    {props.info && (
+      <Kb.WithTooltip tooltip={props.info}>
```
Comment (songgao) [inline shared/chat/blocking/block-modal/index.tsx]: do we maybe want `showOnPressMobile` here?

**Pull request merged**